app/templates/project-name.Gulp/tasks/html.js

```diff
@@ -1,50 +1,55 @@
-import cached from 'gulp-cached';
-import changed from 'gulp-changed';
 import gulp from 'gulp';
-import gutil from 'gulp-util';
-import plumber from 'gulp-plumber';
 import pug from 'gulp-pug';
 import rename from 'gulp-rename';
+import plumber from 'gulp-plumber';
+import changed from 'gulp-changed';
+import cached from 'gulp-cached';
+import colors from 'ansi-colors';
+import log from 'fancy-log';
+import { obj as noop } from 'through2';
 import config from '../config';
 
-function processHTML() {
-    return gulp.src([`${config.html.src}/**/*.pug`, '!**/includes/*.pug'])
-        .pipe(plumber({
-            errorHandler(err) {
-                gutil.log('Filename: ', gutil.colors.bold.red(err.filename));
-                gutil.log('Linenumber: ', gutil.colors.bold.red(err.line));
-                gutil.log('Extract: ', gutil.colors.bold.red(err.msg));
-                gutil.beep();
-                this.emit('end');
-            }
-        }))
-        .pipe(changed((file) => {
-            return file.path.replace('/pug', '');
-        }, { extension: '.html' }))
-        .pipe((global.isWatching && !global.isInclude) ? cached('pug') : gutil.noop())
-        .pipe(pug({
-            data: {
-                site: {
-                    name: config.pkg.name,
-                    namespace: config.namespace,
-                    description: config.pkg.description
-                }
-            },
-            pretty: true
-        }))
-        .pipe(rename((path) => {
-            if (/includes/.test(path.dirname)) {
-                return;
-            }
+const PUG_OPTIONS = {
+    data: {
+        site: {
+            name: config.pkg.name,
+            namespace: config.namespace,
+            description: config.pkg.description
+        }
+    },
+    pretty: true
+};
+
+const PLUMBER_OPTIONS = {
+    errorHandler(err) {
+        log('Filename: ', colors.bold(colors.red(err.filename)));
+        log('Linenumber: ', colors.bold(colors.red(err.line)));
+        log('Extract: ', colors.bold(colors.red(err.msg)));
+        this.emit('end');
+    }
+};
 
-            path.dirname = path.dirname.replace('pug', '');
+function renameTemplate(path) {
+    if (!(/includes/.test(path.dirname))) {
+        path.dirname = path.dirname.replace('pug', '');
 
-            if (path.basename !== 'index') {
-                path.basename = `tpl-${path.basename}`;
-            }
-        }))
+        if (path.basename !== 'index'){
+            path.basename = 'tpl-' + path.basename;
+        }
+    } else {
+        return;
```

Comment:
Is there any reason for having explicit `return`? I mean, it'll return `undefined` anyway in case where it isn't falling into `if` statement. I usually prefer to do an "early return" where possible, so there's less indentation and it reads, but let me know if I'm missing something 😄

Reply:
You are completely right.. actually it should have remained to the change you made.. but I guess I messed it up in my merge. 😭